.github/workflows/backport.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+    paths-ignore:
+      - '_quarto.yml'
+      - 'quarto-materials/*'
+      - '**/.md'
+
+jobs:
+  backport:
+    runs-on: ubuntu-18.04
+    name: Backport
+    steps:
+      - name: Backport
+        uses: tibdex/backport@v1
```

Comment:
Current version seems to be 2.0.3 per https://github.com/tibdex/backport.  But if I read this right (haven't spent a lot of time on it) they suggest to copy their yml file (see prev comment)